src/web/routes/application/children-dob/set-children-in-session.js

```diff
@@ -1,10 +1,10 @@
-const { pickBy } = require('ramda')
+const { pickBy, isNil } = require('ramda')
 const { countKeysContainingString } = require('./count-keys')
 const { DAY_FIELD_SUFFIX } = require('./constants')
 const { isChildEntry } = require('./predicates')
 
 const setChildrenInSessionForGet = (req) => {
-  if (!req.session.hasOwnProperty('children')) {
+  if (isNil(req.session.children)) {
```

Comment:
we need top treat children in the same way (use isNil) in create-request-body when posting to the claimant service